src/components/Display.js

```diff
@@ -1,20 +1,24 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-function Display({ result }) {
+function Display(props) {
+  const { result, live } = props;
   return (
-    <p>
-      {result}
-    </p>
+    <div className="screen">
+      <div className="cal_formula">{live}</div>
+      <div className="cal_total">{result}</div>
+    </div>
   );
 }
```

Comment:
## Optional change 
- [ ] Do you mind destructuring these props from this `Display` function parameters to avoid additional variable declarations like in this case here?
- We can restructure it to appear like the below suggestion. How we have it makes us use extra lines of code and makes the component more verbose.

```suggestion
// This way achieves the same result with less code in a more reusable way
const Display = ({ result, live }) => {
  ...
}
```